src/skia.cpp

```diff
@@ -105,6 +105,24 @@ extern "C"
         CAMLreturn(vColor);
     }
 
+    CAMLprim value
+    caml_SkPath_create()
+    {
+        throw "caml_SkPath_create not implemented yet";
```

Comment:
@manuhornung Left these functions as todos. I couldn't compile without at least defining them in cpp too.